man/rgbfix.1

```diff
@@ -104,6 +105,9 @@ Set the new licensee string
 .Pq Ad 0x144 Ns \(en Ns Ad 0x145
 to a given string.
 If it's longer than 2 chars, it will be truncated, and a warning emitted.
+.It Fl L Ar logo_file , Fl \-logo Ar logo_file
+Specify a logo file to use instead of the official Nintendo logo.
+The file must be a 48-byte 1bpp image (48 pixels wide and 8 pixels tall).
```

Comment:
Calling it an "image" will probably lead people to try feeding that option PNG files.

```suggestion
The file must be 48 bytes of 1bpp tile data (the source image should be 48 pixels wide and 8 pixels tall).
```